plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/configuration/BaseVersionedStorageFormat.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.plugin.services.storage.rocksdb.configuration;
+
+import org.hyperledger.besu.plugin.services.storage.DataStorageFormat;
+
+import java.util.OptionalInt;
+
+public enum BaseVersionedStorageFormat implements VersionedStorageFormat {
+  FOREST_ORIGINAL(DataStorageFormat.FOREST, 1),
+  FOREST_WITH_VARIABLES(DataStorageFormat.FOREST, 2),
+  BONSAI_ORIGINAL(DataStorageFormat.BONSAI, 1),
+  BONSAI_WITH_VARIABLES(DataStorageFormat.BONSAI, 2);
```

Comment:
Do we need to bake in variable and non-variable variants of the datastorage format? The variable change was introduced several versions ago don't think we need to have special logic for versioning this now.

Reply:
The version 2 of each format are here for reference, basically running this version of Besu, as you said, means that each format is, or already was, migrated to the `_WITH_VARIABLES` version.
Does this answer your question?

Reply:
It does. So maybe BONSAI_ORIGINAL isn't needed but think it's still ok.